packages/api/src/__tests__/e2e/mapi/webhook/webhook-handler.ts

```diff
@@ -0,0 +1,76 @@
+import { MetriportMedicalApi } from "@metriport/api-sdk";
+import {
+  DocumentBulkDownloadWebhookRequest,
+  DocumentConversionWebhookRequest,
+  DocumentDownloadWebhookRequest,
+  isConsolidatedWebhookRequest,
+  isDocumentBulkDownloadWebhookRequest,
+  isDocumentConversionWebhookRequest,
+  isDocumentDownloadWebhookRequest,
+} from "@metriport/shared/medical";
+import { Request, Response } from "express";
+import { handleConsolidated } from "./consolidated";
+
+let whKey: string | undefined = undefined;
+
+export async function handleRequest(req: Request, res: Response) {
+  try {
+    if (!isSignatureValid(req)) {
+      console.log(`[WH] ====> Signature verification failed`);
+      throw new Error("Signature verification failed");
+    }
+    console.log(`[WH] ========> Handling request... type: ${req.body?.meta?.type}`);
+    const whRequest = MetriportMedicalApi.parseWebhookResponse(req.body);
+    if (whRequest.meta.type === "ping") {
+      return handlePing(req, res);
+    }
+    if (isConsolidatedWebhookRequest(whRequest)) {
+      return handleConsolidated(whRequest, res);
+    }
+    if (isDocumentConversionWebhookRequest(whRequest)) {
+      return handleDocConversion(whRequest, res);
+    }
+    if (isDocumentDownloadWebhookRequest(whRequest)) {
+      return handleDocDownload(whRequest, res);
+    }
+    if (isDocumentBulkDownloadWebhookRequest(whRequest)) {
+      return handleBulkDownloadUrls(whRequest, res);
```

Comment:
for `handlePing`, `handleDocConversion`, `handleDocDownload`, and `handleBulkDownloadUrls` is the plan to do the same thing as we are doing with with the setter `handleConsolidated` and getter `getConsolidatedData`? If so can you add it to the Outstanding list for the next PR?

Reply:
I didn't add to the outstanding list because those will be done on their specific tickets (v3+ of the [master ticket](https://github.com/metriport/metriport-internal/issues/1634))